### ISSUE TITLE
Toggle's state update is read out twice in Narrator/Edge

### ISSUE THREAD

**Issue by lucas-stanford** [microsoft/fluentui#16809]
### Environment Information
- **Package version(s)**: 7.153.2 (tested on latest, repros' there as well)
- **Browser and OS versions**: Edge/Windows
- Bug: 8800057

### Describe the issue:

Narrator is announcing twice the state of toggle button.

### Please provide a reproduction of the issue in a codepen:

Repro's here: https://fabricweb.z5.web.core.windows.net/oufr/7.158.1/#/examples/toggle

#### Actual behavior:

Narrator is announcing twice the state of toggle button update

#### Expected behavior:

Narrator should announce only once the state of toggle button update

### Documentation describing expected behavior
n/a

**Comment by joschect** [comment]
@lucas-stanford thanks for the submission! I am unable to reproduce this using NVDA, though I can reproduce it with narrator. This is likely an issue with narrator since it looks like we're using the correct aria attributes.
